mailbox/backup/pom.xml

```diff
@@ -43,9 +44,46 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-mailbox-memory</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-mailbox-memory</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
         <dependency>
             <groupId>${james.groupId}</groupId>
             <artifactId>apache-james-mailbox-store</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-mailbox-store</artifactId>
```

Comment:
Why do we need this dependency for?

Better not to have it....

Reply:
We need for the BlobStore, and the BlobId. ( first i used the blobid from the mailbox module, it was a mistake fixed in the last iteration of JAMES-2681)

Reply:
I thought itwas for the Mailbox model - see below proposals. 

IMO depending on mailbox-store in tests in kinda ok, but in prod code in new module this should be considered being a code smell.

Reply:
I think the blobstore is one of the main component of the backup api. No ?
I do not see your proposals ?

Reply:
Doesn't the blob-store comes from blob-api ?

This component has nothing to do with mailbox-store....

Reply:
we need the StoreMessageManager  to get the Mailbox.